react/nextJsQuestions.md

```diff
@@ -0,0 +1,37 @@
+# Next.js Interview Questions
+
+## General Questions:
+
+[What is Next.js, and how does it differ from traditional React applications?](https://nextjs.org/docs/getting-started/introduction)
+
+[Explain the concept of server-side rendering (SSR) in Next.js](https://nextjs.org/docs/basic-features/data-fetching)
+
+[How does Next.js handle client-side routing?](https://nextjs.org/docs/navigation/introduction)
+
+[What are the benefits of using Next.js for building React applications?](https://nextjs.org/learn/basics/why-use-nextjs)
+
+[What is the purpose of the `getInitialProps` function in Next.js? How is it different from using `getStaticProps` or `getServerSideProps`?](https://nextjs.org/docs/api-reference/data-fetching/getInitialProps)
```

Comment:
replace to https://nextjs.org/docs/pages/api-reference/functions/get-initial-props

Reply:
https://nextjs.org/docs/pages/api-reference/functions/get-server-side-props
https://nextjs.org/docs/pages/api-reference/functions/get-static-props